test/CMakeLists.txt

```diff
@@ -147,25 +147,12 @@ if(PYTHONINTERP_FOUND)
       set(VARIANT_SDK "${SWIFT_SDK_${SDK}_PATH}")
 
       # A directory where to put the xUnit-style XML test results.
-      set(swift_test_results_dir
+      set(SWIFT_TEST_RESULTS_DIR
```

Comment:
It's no longer a local; I need to get it inside `lit`. Though, if there's a better way to get to that directory, I'd be happy to use it!